src/Fantomas.Core/ASTTransformer.fs

```diff
@@ -2016,16 +2016,39 @@ let mkSynRationalConst (creationAide: CreationAide) rc =
         | SynRationalConst.Integer(i, range) ->
             stn (creationAide.TextFromSource (fun () -> string i) range) range
             |> RationalConstNode.Integer
-        | SynRationalConst.Rational(numerator, numeratorRange, denominator, denominatorRange, range) ->
+
+        | SynRationalConst.Paren(SynRationalConst.Rational(numerator,
+                                                           numeratorRange,
+                                                           divRange,
+                                                           denominator,
+                                                           denominatorRange,
+                                                           _),
+                                 range) ->
+            let openingParen =
+                let r =
+                    withEnd (Position.mkPos range.Start.Line (range.StartRange.StartColumn + 1)) range.StartRange
+
+                stn (creationAide.TextFromSource (fun () -> "(") r) r
```

Comment:
Can this ever be anything else than `(`? It feels like overkill to grab this from the source text.

Reply:
Currently it can only be `(`. I followed the idiom seen elsewhere to blindly. I'll adjust.

Reply:
`creationAide.TextFromSource` is mostly used for user-defined code.
There are plenty of examples where we do `stn "x"` where `x` is a hardcoded thing.
https://github.com/fsprojects/fantomas/blob/646dcc9f9b28373247695abbbe79474819b80403/src/Fantomas.Core/ASTTransformer.fs#L74-L79

Reply:
Yes, sorry. I looked at the wrong example.